lib/helper.js

```diff
@@ -5,6 +5,21 @@ const hoek = require('@hapi/hoek');
 const dayjs = require('dayjs');
 const TEMPLATE_NAME_REGEX_WITH_NAMESPACE = schema.config.regex.FULL_TEMPLATE_NAME_WITH_NAMESPACE;
 const formatDate = dateTime => dayjs(dateTime).format('YYYY-MM-DD');
+// SCM handles only SUCCESS, PENDING & FAILURE status
+const SCM_STATE_MAP = {
+    ABORTED: 'FAILURE',
+    CREATED: 'PENDING',
+    FAILURE: 'FAILURE',
+    QUEUED: 'PENDING',
+    RUNNING: 'PENDING',
+    SUCCESS: 'SUCCESS',
+    BLOCKED: 'PENDING',
+    UNSTABLE: 'FAILURE',
```

Comment:
UNSTABLE status means the next build will not be triggered, but does not mean failure; maybe PENDING would be more equivalent
```suggestion
    UNSTABLE: 'PENDING',
```